server/lobbyconnection.py

```
@@ -604,14 +604,20 @@ def decodeUniqueId(self, serialized_uniqueid):
 
             return m.hexdigest(), (UUID, mem_SerialNumber, DeviceID, Manufacturer, Name, ProcessorId, SMBIOSBIOSVersion, SerialNumber, VolumeSerialNumber)
         except Exception as ex:
-            self._logger.exception(ex)
```

Please leave in logging for any catch-all exception handlers.